puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-x-develop-release.groovy

```diff
@@ -15,33 +15,12 @@ pipeline {
     }
 
     stages {
-        stage('Setup Environment') {
+        stage('Copy Source') {
```

Comment:
Maybe this file should now live in release/package directory?

Reply:
Do you mean `pipelines/release/package/foreman-x-develop.groovy`?